### PR TITLE
Copy edit 5.16 API articles, A through F

### LINKED ISSUE #1975: Copy-edit Go API articles (A through F)
- Fix typos
- Use numbers for link references
- Fix broken links
- Add trailing slash to all internal links
- Apply line break standard (one sentence per line)
- Confirm all code examples use the correct api_version
- Standardize in-doc heading capitalization, remove gerunds, and correct heading levels
- Add and complete top-of-page TOCs
- Make other style corrections as needed

## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Affected Docs Pages
All articles in the Go 5.16 API section whose titles start with A through F

## Context
Copy edit cleanup

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Fixed typos
- Replaced text links with numbered link references
- Fixed broken links
- Added trailing slash to all internal links
- Applied line break standard (one sentence per line)
- Standardized in-doc heading capitalization, removed gerunds, and corrected heading levels
- Added and completed top-of-page TOCs
- Other style corrections as needed

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/1975

## Review Instructions
I made a lot of changes to the Events doc -- there were some missing specs and headings

### CODE REVIEW

File: content/sensu-go/5.16/api/auth.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Authentication API"
-description: "The authentication API provides HTTP access to test user credentials. Here’s a reference for the authentication API in Sensu Go, including an example for how to query the authentication API to determine whether credentials are valid. Read on for the full reference."
+description: "The Sensu authentication API provides HTTP access to test user credentials. This reference includes an example of how to query the authentication API to determine whether credentials are valid. Read on for the full reference."
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/auth.md]:
```suggestion
description: "The Sensu authentication API provides HTTP access to test user credentials and use such credentials to obtain access tokens. This reference includes an example of how to query the authentication API to determine whether credentials are valid. Read on for the full reference."
```
I'm struggling to word this better, but the existing description seems to focused on testing credentials, doesn't suggest you can actually use endpoints in this API to authenticate.

File: content/sensu-go/5.16/api/cluster-roles.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Cluster roles API"
 linkTitle: "Cluster Roles API"
-description: "The cluster role API provides HTTP access to cluster role data. Here’s a reference for the cluster roles API in Sensu Go, including examples for returning lists of cluster roles, creating Sensu cluster roles, and more. Read on for the full reference."
+description: "The Sensu cluster roles API provides HTTP access to cluster role data. This refernece includes examples for returning lists of cluster roles, creating Sensu cluster roles, and more. Read on for the full reference."
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/cluster-roles.md]:
```suggestion
description: "The Sensu cluster roles API provides HTTP access to cluster role data. This reference includes examples for returning lists of cluster roles, creating Sensu cluster roles, and more. Read on for the full reference."
```

File: content/sensu-go/5.16/api/cluster.md [hunk 3]
```diff
@@ -188,7 +190,7 @@ HTTP/1.1 204 No Content
 
 /cluster/ members/:member (DELETE) |
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/cluster.md]:
```suggestion
/cluster/members/:member (DELETE) | 
```

File: content/sensu-go/5.16/api/filters.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Filters API"
-description: "The filters API provides HTTP access to filter data. Here’s a reference for the filters API in Sensu Go, including examples for returning lists of filters, creating Sensu filters, and more. Read on for the full reference."
+description: "The Sensu filters API provides HTTP access to filter data. This reference includes examples for returning lists of filters, creating Sensu filters, and more. Read on for the full reference."
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/filters.md]:
```suggestion
description: "The Sensu filters API provides HTTP access to event filter data. This reference includes examples for returning lists of filters, creating Sensu filters, and more. Read on for the full reference."
```

File: content/sensu-go/5.16/api/filters.md [hunk 5]
```diff
@@ -24,8 +24,7 @@ The `/filters` API endpoint provides HTTP GET access to [filter][1] data.
 
 #### EXAMPLE {#filters-get-example}
 
-The following example demonstrates a request to the `/filters` API, resulting in
-a JSON Array containing [filter definitions][1].
+The following example demonstrates a request to the `/filters` API, resulting in a JSON array that contains [filter definitions][1].
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/filters.md]:
```suggestion
The following example demonstrates a request to the `/filters` API, resulting in a JSON array that contains [event filter definitions][1].
```

File: content/sensu-go/5.16/api/filters.md [hunk 6]
```diff
@@ -88,9 +87,13 @@ output         | {{< highlight shell >}}
 
 ### `/filters` (POST)
 
+The `/filters` API endpoint provides HTTP POST access to create a filter.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/filters.md]:
```suggestion
The `/filters` API endpoint provides HTTP POST access to create an event filter.
```

File: content/sensu-go/5.16/api/filters.md [hunk 7]
```diff
@@ -88,9 +87,13 @@ output         | {{< highlight shell >}}
...
+
+#### API Specification {#filters-post-specification}
+
 /filters (POST) | 
 ----------------|------
-description     | Create a Sensu filter.
+description     | Creates a Sensu filter.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/filters.md]:
```suggestion
description     | Creates a Sensu event filter.
```

File: content/sensu-go/5.16/api/filters.md [hunk 8]
```diff
@@ -142,7 +143,7 @@ curl http://127.0.0.1:8080/api/core/v2/namespaces/default/filters/state_change_o
 
 /filters/:filter (GET) | 
 ---------------------|------
-description          | Returns a filter.
+description          | Returns the specified filter.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/filters.md]:
```suggestion
description          | Returns the specified event filter.
```

File: content/sensu-go/5.16/api/filters.md [hunk 9]
```diff
@@ -164,11 +165,13 @@ output               | {{< highlight json >}}
 
 ### `/filters/:filter` (PUT) {#filtersfilter-put}
 
+The `/filters/:filter` API endpoint provides HTTP PUT access to create or update a filter.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/filters.md]:
```suggestion
The `/filters/:filter` API endpoint provides HTTP PUT access to create or update an event filter.
```

File: content/sensu-go/5.16/api/filters.md [hunk 10]
```diff
@@ -164,11 +165,13 @@ output               | {{< highlight json >}}
...
+
 #### API Specification {#filtersfilter-put-specification}
 
 /filters/:filter (PUT) | 
 ----------------|------
-description     | Create or update a Sensu filter.
+description     | Creates or updates the specified Sensu filter.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/filters.md]:
```suggestion
description     | Creates or updates the specified Sensu event filter.
```

File: content/sensu-go/5.16/api/filters.md [hunk 11]
```diff
@@ -189,10 +192,11 @@ response codes  | <ul><li>**Success**: 201 (Created)</li><li>**Malformed**: 400
 
 ### `/filters/:filter` (DELETE) {#filtersfilter-delete}
 
-The `/filters/:filter` API endpoint provides HTTP DELETE access to delete a filter from Sensu given the filter name.
+The `/filters/:filter` API endpoint provides HTTP DELETE access to delete a filter from Sensu (specified by the filter name).
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/filters.md]:
```suggestion
The `/filters/:filter` API endpoint provides HTTP DELETE access to delete an event filter from Sensu (specified by the filter name).
```

File: content/sensu-go/5.16/api/filters.md [hunk 12]
```diff
@@ -189,10 +192,11 @@ response codes  | <ul><li>**Success**: 201 (Created)</li><li>**Malformed**: 400
...
+
+#### EXAMPLE {#filtersfilter-delete-example}
 
-### EXAMPLE {#filtersfilter-delete-example}
-The following example shows a request to delete the filter `production-only`, resulting in a successful HTTP 204 No Content response.
+The following example shows a request to delete the filter `production-only`, resulting in a successful HTTP `204 No Content` response.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/filters.md]:
```suggestion
The following example shows a request to delete the event filter `production-only`, resulting in a successful HTTP `204 No Content` response.
```

File: content/sensu-go/5.16/api/filters.md [hunk 13]
```diff
@@ -206,8 +210,9 @@ HTTP/1.1 204 No Content
 
 /filters/:filter (DELETE) | 
 --------------------------|------
-description               | Removes a filter from Sensu given the filter name.
+description               | Removes the specified filter from Sensu.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/filters.md]:
```suggestion
description               | Removes the specified event filter from Sensu.
```

File: content/sensu-go/5.16/api/overview.md [hunk 14]
```diff
@@ -14,48 +14,55 @@ menu:
...
 - `{version}` is the API version: `v2`.
-- `{namespace}` is the namespace name. The examples in these API docs use the `default` namespace. The Sensu API requires that the authenticated user have the correct access permissions for the namespace specified in the URL. If the authenticated user has the correct cluster-wide permissions, you can leave out the `/namespaces/{namespace}` portion of the URL to access Sensu resources across namespaces. See the [RBAC reference][3] for more information about configuring Sensu users and access controls.
+- `{namespace}` is the namespace name.
+The examples in these API docs use the `default` namespace.
+The Sensu API requires the authenticated user to have the correct access permissions for the namespace specified in the URL.
+If the authenticated user has the correct cluster-wide permissions, you can leave out the `/namespaces/{namespace}` portion of the URL to access Sensu resources across namespaces.
+See the [RBAC reference][3] for more information about configuring Sensu users and access controls.
+
+_**NOTE**: The [authentication API][12] and [authentication providers API][15] do not follow this standard URL format._
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/overview.md]: I believe the /health endpoint does not follow the standard URL format.

**Review by cwjohnston — CHANGES_REQUESTED** [review]
I've suggested adding "event" to some of the places we talk about filters because I think `filter` is becoming an overloaded term in Sensu  -- assets can have filters, handlers can have (event) filters.

there may be more places in the filter api doc which need that treatment, or we can consider such a change separately from this effort. ¯\_(ツ)_/¯

File: content/sensu-go/5.16/api/auth.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Authentication API"
-description: "The authentication API provides HTTP access to test user credentials. Here’s a reference for the authentication API in Sensu Go, including an example for how to query the authentication API to determine whether credentials are valid. Read on for the full reference."
+description: "The Sensu authentication API provides HTTP access to test user credentials and use such credentials to obtain access tokens. This reference includes an example of how to query the authentication API to determine whether credentials are valid. Read on for the full reference."
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/auth.md]:
```suggestion
description: "The Sensu authentication API provides HTTP access to test whether user credentials are valid and use these credentials to obtain access tokens. Read on for the full reference."
```

Comment (hillaryfraley) [reply content/sensu-go/5.16/api/auth.md]: wdyt of this?

File: content/sensu-go/5.16/api/overview.md (hunk 14 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/overview.md]:
```suggestion
_**NOTE**: The [authentication API][12], [authentication providers API][15], and [health API][5] do not follow this standard URL format._
```

**Comment by hillaryfraley** [comment]
@cwjohnston I read back through the filter doc and added the "event" qualifier in a few more spots.

File: content/sensu-go/5.16/api/overview.md [hunk 16]
```diff
@@ -14,48 +14,55 @@ menu:
 - [Data format](#data-format)
 - [Versioning](#versioning)
 - [Access control](#access-control)
+  - [Authentication quickstart](#authentication-quickstart)
 - [Pagination](#pagination)
 - [Filtering](#filtering)
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/overview.md]:
```suggestion
- [Response Filtering](#response-filtering)
```

File: content/sensu-go/5.16/api/overview.md [hunk 17]
```diff
@@ -238,26 +251,33 @@ Content-Type: application/json
 
 ## Filtering
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/overview.md]:
```suggestion
## Response Filtering
```

File: content/sensu-go/5.16/api/overview.md [hunk 18]
```diff
@@ -238,26 +251,33 @@ Content-Type: application/json
...
 
-**COMMERCIAL FEATURE**: Access API filtering in the packaged Sensu Go distribution. For more information, see the [getting started guide][8].
+**COMMERCIAL FEATURE**: Access API filtering in the packaged Sensu Go distribution.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/overview.md]:
```suggestion
**COMMERCIAL FEATURE**: Access API response filtering in the packaged Sensu Go distribution.
```

File: content/sensu-go/5.16/api/overview.md [hunk 19]
```diff
@@ -238,26 +251,33 @@ Content-Type: application/json
...
+For more information, see [Get started with commercial features][8].
 
-The Sensu API supports filtering for all GET endpoints that return an array. You can filter resources based on their labels with a label selector using the `labelSelector` query parameter and on certain pre-determined fields with a field selector using the `fieldSelector` query parameter.
+The Sensu API supports filtering for all GET endpoints that return an array.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/overview.md]:
```suggestion
The Sensu API supports response filtering for all GET endpoints that return an array.
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
In order to sufficiently differentiate from event filtering, I'd like us to frame API filtering as "response filtering".

**Pull request merged**